src/openrct2/core/File.cpp

```diff
@@ -27,7 +27,9 @@ namespace File
     {
         fs::path file = u8path(path);
         log_verbose("Checking if file exists: %s", std::string(path).c_str());
-        return fs::exists(file);
+        std::error_code ec;
+        const auto result = fs::exists(file, ec);
+        return result && ec.value() == 0;
```

Comment:
`std::error_code` does have an `operator bool()` implementation, so technically you could do:
```suggestion
        return result && !ec;
```

I'll leave it up to you to decide. If you follow my suggestion, make sure you update on the rest of the diff.

Reply:
You know me, I like being more explicit ;-)

Reply:
Yeah, I thought so :PP